pkg/analysis/passes/sourcecode/sourcecode.go

```diff
@@ -0,0 +1,87 @@
+package sourcecode
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/tailscale/hujson"
+
+	"github.com/grafana/plugin-validator/pkg/analysis"
+	"github.com/grafana/plugin-validator/pkg/analysis/passes/metadata"
+)
+
+var (
+	sourceCodeNotProvided      = &analysis.Rule{Name: "source-code-not-provided", Severity: analysis.Warning}
+	sourceCodeNotFound         = &analysis.Rule{Name: "source-code-not-found", Severity: analysis.Error}
+	sourceCodeVersionMissMatch = &analysis.Rule{Name: "source-code-version-missmatch", Severity: analysis.Error}
```

Comment:
Typo: this should be spelled as "mismatch", with one "s" (both in the variable name and in the `Name` string)